configure.sh

```diff
@@ -88,45 +59,22 @@ sudo pip install -U PyYAML \
                     exifread \
                     gpxpy \
                     xmltodict
-if [ $? -ne 0 ] 
-then
-    echo -e "\e[1;31mERROR: \e[39mError when Installing OpenSfM Dependencies\e[0m"
-    exit 1
-fi
 
-## Installing Ecto Requisites
-echo -e "\e[1;34mInstalling Ecto Dependencies\e[0;39m"
+echo "Installing Ecto Dependencies"
 sudo pip install -U catkin-pkg
-sudo apt-get install python-empy \
+sudo apt-get install -y python-empy \
                      python-nose \
                      python-pyside -y -qq
-if [ $? -ne 0 ] 
-then
-    echo -e "\e[1;31mERROR: \e[39mError when Installing Ecto Dependencies\e[0m"
-    exit 1
-fi
 
-## Installing OpenDroneMap Requisites
-echo -e "\e[1;34mInstalling OpenDroneMap Dependencies\e[0;39m"
-sudo apt-get install python-pyexiv2 \
+echo "Installing OpenDroneMap Dependencies"
+sudo apt-get install -y python-pyexiv2 \
                      python-scipy \
                      jhead \
                      liblas-bin -y -qq
-if [ $? -ne 0 ] 
-then
-    echo -e "\e[1;31mERROR: \e[39mError when Installing OpenDroneMap Dependencies\e[0m"
-    exit 1
-fi
-
-## Get sys vars
-NUM_CORES=`grep -c processor /proc/cpuinfo`
-
-## Add SuperBuild path to the python path
-export PYTHONPATH=$PYTHONPATH:`pwd`/SuperBuild/install/lib/python2.7/dist-packages:`pwd`/SuperBuild/src/opensfm
 
-## Compile SuperBuild
+echo "Compiling SuperBuild"
 cd SuperBuild
 mkdir -p build && cd build
-cmake .. && make -j${NUM_CORES}
+cmake .. && make -j8
```

Comment:
I actually just noticed that many of the things in configure.sh are already added in packages.Dockerfile >:( I think the optimal fix is to avoid using configure.sh in the Docker use case. Looking into a clean way to do this.